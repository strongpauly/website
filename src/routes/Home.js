import React, { Component } from 'react';

export default class Home extends Component {

  render() {
    return <div id="homeContainer" className="container"><p>This is a portfolio of projects used in investigations into technologies by London based, full stack developer, Paul Potsides.</p>
      <p>Currently, this is mainly client side and react based projects, but I will hopefully be expanding this into some Java or Node based middle tier projects too.</p>
      <p>This site was built using react, react-bootstrap and react-router.  It&apos;s source can be found <a href="https://www.github.com/strongpauly/website">here</a>
      </p>
    </div>;
  }

}
