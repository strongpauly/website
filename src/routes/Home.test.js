import React from 'react';
import Home from './Home';
import {shallow} from 'enzyme';

/* eslint-env jest */

describe('Home', () => {
  it('renders without crashing', () => {
    const app = shallow(<Home/>);
    expect(app).toMatchSnapshot();
  });

  it('creates a #homeContainer', () => {
    const app = shallow(<Home/>);
    expect(app.find('#homeContainer')).toHaveLength(1);
  });
});
