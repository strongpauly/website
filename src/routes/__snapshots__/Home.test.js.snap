// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
<div
  id="homeContainer"
>
  <p>
    This is a portfolio of projects used in investigations into technologies by London based, full stack developer, Paul Potsides.
  </p>
  <p>
    Currently, this is mainly client side and react based projects, but I will hopefully be expanding this into some Java or Node based middle tier projects too.
  </p>
  <p>
    This site was built using react, react-bootstrap and react-router.  It's source can be found 
    <a
      href="https://www.github.com/strongpauly/website"
    >
      here
    </a>
  </p>
</div>
`;
